### ISSUE TITLE
Default values for enum arguments is not according to the spec

### ISSUE THREAD

**Issue by praveensvsrk** [gabotechs/graphqxl#54]
I am seeing incorrect behaviour when using default values for enum arguments. I get parsing error during conversion. I have demonstrated this using a small example below.

```gql
enum Language {
    DE
    FR
}

type Query {
    toEnglish(from: Language! = DE): String
}
```

This is a valid schema, but graphqxl fails to parse it. Notice the error while parsing the default value for the argument `from`.

![image](https://github.com/gabotechs/graphqxl/assets/10628197/33c4d498-a809-429e-9601-2a63539eb8a0)

however, string literal as value is getting accepted, which shouldn't be the case.

![image](https://github.com/gabotechs/graphqxl/assets/10628197/a38846fe-ea07-479e-9826-ad7af61a424d)

ref: [Oct 2021 spec](https://spec.graphql.org/October2021/#EnumValue)

**Comment by jasonav** [comment]
@gabotechs Can you take a look at this? It is a blocker issue for us.

**Comment by gabotechs** [comment]
Taking a look now...

**Comment by gabotechs** [comment]
It was an easy fix, it's already released under the `0.40.1` tag. Feel free to reopen if you find any issue.

**Comment by jasonav** [comment]
Thank you!!
